packages/playwright-test/src/reporters/base.ts

```diff
@@ -253,6 +257,12 @@ export class BaseReporter implements ReporterV2 {
       console.log(colors.yellow('  Consider splitting slow test files to speed up parallel execution'));
   }
 
+  private _printMaxFailuresReached(skipped: number) {
+    if (this._failureCount < this.config.maxFailures)
+      return;
+    console.log(colors.yellow(`  Maximum number of allowed failures (${this.config.maxFailures}) reached. ${skipped} remaining tests were skipped.`));
```

Comment:
This is not accurate, because `skipped` includes those tests that were marked as "skipped" as well. Let's not print this number :)

I'd recommend something like `Testing stopped early after 23 maximum allowed failures`.